.github/workflows/test.yml

```diff
@@ -9,6 +9,21 @@ on:
       - '.gitignore'
 
```

Comment:
🔧 Ensure that the 'requirements.txt' file is up-to-date with all necessary dependencies for the tests.